ui/apps/platform/src/Containers/NetworkGraph/utils/flowUtils.ts

```diff
@@ -303,3 +303,29 @@ export function createFlowsFromGroupedDiffFlows(
     });
     return flows;
 }
+
+export function getNumAllowedEgressFlows(nodes: CustomNodeModel[]): number {
+    const extraneousEgressNode = nodes.find((node) => {
+        return node.id === 'extraneous-egress-flows';
+    });
+    if (!extraneousEgressNode || extraneousEgressNode.visible === false) {
+        return 0;
+    }
+    const numAllowedEgressFlows =
+        extraneousEgressNode?.data.type === 'EXTRANEOUS' ? extraneousEgressNode?.data.numFlows : 0;
+    return numAllowedEgressFlows;
+}
+
+export function getNumAllowedIngressFlows(nodes: CustomNodeModel[]): number {
```

Comment:
nit: Do these functions need to have "Extraneous" in the name, instead of "Allowed"?

Reply:
Yea I guess they should include that